bql/semantic/expression_test.go

```diff
@@ -181,26 +181,26 @@ func TestBooleanEvaluationNode(t *testing.T) {
 	for _, entry := range testTable {
 		got, err := entry.eval.Evaluate(table.Row{})
 		if !entry.err && err != nil {
-			t.Errorf("failed to evaluate op %q for %v with error %v", entry.eval.(*booleanNode).op, entry.eval, err)
+			t.Errorf("failed to evaluate op %q for %v with error: %v", entry.eval.(*booleanNode).op.String(), entry.eval, err)
 		}
 		if want := entry.want; got != want {
-			t.Errorf("failed to evaluate op %q for %v; got %v, want %v", entry.eval.(*booleanNode).op, entry.eval, got, want)
+			t.Errorf("failed to evaluate op %q for %v; got %v, want %v", entry.eval.(*booleanNode).op.String(), entry.eval, got, want)
 		}
 	}
 }
 
-func buildLiteralOrDie(textLiteral string) *literal.Literal {
+func mustBuildLiteral(textLiteral string) *literal.Literal {
 	lit, err := literal.DefaultBuilder().Parse(textLiteral)
 	if err != nil {
-		panic(fmt.Sprintf("Could not parse text literal got err: %v", err.Error()))
+		panic(fmt.Sprintf("could not parse text literal %q, got error: %v", textLiteral, err))
 	}
 	return lit
 }
 
-func newNodeFromStringOrDie(nodeType, nodeID string) *node.Node {
+func mustBuildNodeFromStrings(nodeType, nodeID string) *node.Node {
```

Comment:
Optional: Do you build a Node from anything else? If not, I'd remove the `FromStrings` suffix and simply call this function `mustBuildNode`.

Reply:
It is also possible to create a node from the parameters `(t *Type, id *ID)`, so I will keep this function being called `mustBuildNodeFromStrings ` for clarity for now.